Fixed some inaccuracies and typos. (#97)
Joystream/helpdesk: proposals/README.md

> When the `Council` was first introduced, it was only able to vote on whether or not a runtime upgrade should be implemented. This  rarely occurred, making the Council a rather insignificant part of the platform.

Reviewer: ```suggestion
When the `Council` was first introduced, it was only able to vote on whether or not a runtime upgrade should be implemented. This very rarely occurred, making the Council a rather insignificant part of the platform.
```

> As the `Council` is meant to be the executive arm of the Joystream governance system, we are adding a proposal system that will allow the users to control most of the day to day decision making on the platform.

Reviewer: ```suggestion
As the `Council` is intended to be the executive arm of the Joystream governance system, we have now developed a proposal system that allows users to control most of the day-to-day decision making on the platform.
```

- 0 `Slash`
- 0 `Abstain`

> At this point, the `Approval Quorum` parameter is fulfilled (100%>75% approval), but there are still to few votes cast to fulfill the `Approval Threshold` (at 30%<50%).

Reviewer: ```suggestion
At this point, the `Approval Quorum` parameter is fulfilled (100%>75% approval), but there are still too few votes cast to fulfil the `Approval Threshold` (at 30%<50%).
```
